web/src/pages/Vulnerability/VulnerabilityView.jsx

```diff
@@ -52,38 +56,35 @@ export function VulnerabilityView(props) {
           </List>
         </Card>
       </Box>
-      {/* Topic misp tags */}
+      {/* Vuln cve ids */}
```

Comment:
```suggestion
      {/* Vuln cve id */}
```
CVE IDはVuln1件につき0または1件のみ

Reply:
修正しました。